packages/api/src/controllers/stream.ts

```diff
@@ -1097,20 +1068,7 @@ app.put(
 
     if (!stream.isActive || streamNuked) {
       const ingest = await getIngestBase(req);
-      setImmediate(async () => {
-        try {
-          await triggerCatalystPullStart(
-            stream,
-            getHLSPlaybackUrl(ingest, stream)
-          );
-        } catch (err) {
-          logger.error("Error triggering catalyst pull start err=", err);
-        }
-      });
-
-      if (waitActive === "true") {
-        stream = await pollWaitStreamActive(req, stream.id);
-      }
+      await triggerCatalystPullStart(stream, getHLSPlaybackUrl(ingest, stream));
```

Comment:
so in practice we always wait for the stream to start, is that right?

Reply:
yes if the manifest request has returned then the stream has started and ready for playback